community/lucene-index/src/test/java/org/neo4j/kernel/api/impl/index/DatabaseIndexIntegrationTest.java

```diff
@@ -76,6 +78,10 @@
     public final RuleChain ruleChain = RuleChain.outerRule( testDirectory ).around( repeatRule )
             .around( fileSystemRule );
 
+    public final VerboseTimeout timeout = VerboseTimeout.builder()
```

Comment:
you need to include this rule into a chain of rules; also if we use this rule for timeouts we should not use separate per test timeouts